playbooks/portals-set-allowance-price-controls.yml

```diff
@@ -20,10 +20,12 @@
     - name: Fail if you are targeting all dev and prod webportals
       include_tasks: tasks/host-limit-check.yml
 
-    # Check/install Ansible prerequisities
+    # Check/install Ansible prerequisites
     - name: Include preparing portal prerequisities
       include_tasks: tasks/portals-prepare.yml
 
     # Set allowance
     - name: Set allowance
-      command: "docker exec sia siac renter setallowance --max-storage-price {{ max_storage_price }}"
\ No newline at end of file
+      ansible.builtin.command: |
+        docker exec sia siac renter setallowance --max-storage-price {{ max_storage_price }} --max-contract-price {{ max_contract_price }} --max-sector-access-price {{ max_sector_access_price }}
+      register: set_allowance_result
```

Comment:
This is not used anywhere. Do we need it?

Reply:
well....you added it just now :-) 
So if you aren't using it then you can delete it.

Reply:
I did some "creative borrowing" from Fil's code without fully understanding it. ;)
I'll just double down on that and add the failure check he has.